src/AppRouter.js

```diff
@@ -130,17 +130,17 @@ class AppRouter extends Component {
     var valueFromURL = getTakeTourFromURL();
     var valueFromStorage = window.localStorage.getItem(TOUR_STORAGE_KEY);
     //----- value passed via url should take precedence over one in storage if provided, and should overwrite local storage value -------
-    var showTour = valueFromURL || valueFromStorage;
-    showTour = showTour === "false" ? false : true;
+    valueFromURL = valueFromURL === "true" ? true : false;
+    if (valueFromURL) return this.props.setTourState(valueFromURL);
 
-    window.localStorage.setItem(TOUR_STORAGE_KEY, showTour);
-    this.props.setTourState(showTour);
+    valueFromStorage = valueFromStorage === "false" ? false : true;
```

Comment:
question - can the value returned from storage come back as a boolean false, rather than a string "false"?